extensions/arc/package.json

```diff
@@ -1076,7 +1076,7 @@
     "@types/sinon": "^9.0.4",
     "@types/uuid": "^8.3.0",
     "@types/yamljs": "^0.2.31",
-    "azdata-test": "^1.0.0",
+    "azdata-test": "^1.1.0",
```

Comment:
Can you go through and update all the other extensions as well? Should try to keep these in sync generally.